packages/server/src/server.ts

```diff
@@ -235,35 +228,70 @@ const redirectToLoginServer = (req: express.Request, res: express.Response) => {
   return false;
 };
 
+/*
+ * If a session exists, return currently logged in account information.
+ */
 app.get(
```

Comment:
Just sort of a general comment, perhaps it would be nice to split all these routes out to a different file(s) and export the router from there as to streamline the server file a bit.

Reply:
I agree, I created this [tracking bug](https://github.com/GoogleCloudPlatform/wombat-dressing-room/issues/125) I would like to improve coverage in `server.ts` before we tear the file apart too much.